shell/platform/windows/keyboard_unittests.cc

```diff
@@ -544,9 +544,11 @@ class KeyboardTest : public WindowsTest {
 // Define compound `expect` in macros. If they're defined in functions, the
 // stacktrace wouldn't print where the function is called in the unit tests.
 
-#define EXPECT_CALL_IS_EVENT(_key_call, ...)         \
-  EXPECT_EQ(_key_call.type, KeyCall::kKeyCallOnKey); \
-  EXPECT_EVENT_EQUALS(_key_call.key_event, __VA_ARGS__);
```

Comment:
Visual Studio doesn't expand `__VA_ARGS__` well. Instead of "expanding" the `__VA_ARGS` into multiple arguments, it places them all into a single argument. This causes `EXPECT_EVENT_EQUALS` to have unexpected commas while in Visual Studio.